test/performance/README.md

```diff
@@ -49,6 +49,11 @@ if err != nil {
 }
 ```
 
+## Zipkin trace
+
+Knative adds zipkin traces for requests made by `SpoofingClient` with HTTP server error responses.
```

Comment:
The `setupZipkinTracing` function seems to enable Zipkin for all API calls. I am not sure what is mean't by the "with HTTP server error responses.

Also curious about two things:
1. What is the performance impact of tracing for all calls? (i.e. Are we accurately measuring production performance?)
1. How do I look at a trace that comes out of zipkin?

Reply:
The `setupZipkinTracing` sets up zipkin tracing and run `logZipkinTrace` after the response is retrieved. In the `logZipkinTrace` method, it only logs the traces for HTTP server errors. 

```
func (sc *SpoofingClient) logZipkinTrace(spoofResp *Response) {
	if !zipkin.ZipkinTracingEnabled || spoofResp.StatusCode < http.StatusInternalServerError || spoofResp.StatusCode >= 600 {
		return
	}
        ....
```
Source: https://github.com/knative/pkg/blob/master/test/spoof/spoof.go#L263-L265

1. Currently, there would be performance impact for tracing. In case where there's a server error, there would be at least 5 seconds extra latency. I have PR https://github.com/knative/pkg/pull/319 to increase it since even with the 5 second wait time, I've seen traces not being published. 
I also created an issue to batch fetch the zipkin logs at the end of the test https://github.com/knative/pkg/issues/317 to make it less impactful on performance during the test.  

1.  The traces would get printed out to the raw log. Here's a sample trace for `TestObservedConcurrency` after I commented out the server error check above. 
https://drive.google.com/file/d/17uE9fBTwcEx3W2vWDi0Ib26bVZ2boH7_/view?usp=sharing

Reply:
Can you add this information to the README? It seems useful to understand that tracing is on every request, but only logged for errors as well as how to access and interpret the log messages.

Reply:
Thanks for raising the confusion. Updated README to clarify when the zipkin traces get logged. 

Per offline discussion, accessing and interpreting the log messages will be done separately after measuring the latency impact of enabling zipkin for all request. This PR focuses on enabling an optional test flag to turn on zipkin logging for the perf test.